openfasoc/generators/ldo-gen/blocks/sky130hvl/ldo_domain_insts.txt

```diff
@@ -0,0 +1,53 @@
+cmp1
+pmos_1
+pmos_2
+pt_array_unit1
```

Comment:
This file is hard-coded? it should be updated by the `ldo-gen.py` for other configurations based on the Verilog output.
`Basically test.json (ILOAD) => new verilog (numberofPT) => new (area, ldo_domain_list, etc..)
`